lib/model/utils.dart

```diff
@@ -0,0 +1,12 @@
+/// Util:
+///   - a static class that provides
+///     various utilities
+class Utils {
+  static Iterable<U> mapIndexed<T, U>(
```

Comment:
Really you don't need this method at all. It looks like it was copy and pasted from the `collection` library. Just add the library as a dependency and import it, instead of copying the code.

Reply:
https://pub.dev/packages/collection

Reply:
https://pub.dev/packages/collection